_docs/_latest/concepts/architecture.md

```diff
@@ -1,8 +1,42 @@
 ---
-title: Architecture
-order: 100
+title: Forseti Architecture
```

Comment:
I think this is Okay being just Architecture.

Reply:
Done.